tests/asn1-reference/uptane/ipuptane_message.asn1

```diff
@@ -0,0 +1,112 @@
+IpUptane DEFINITIONS ::= BEGIN
+        AKIpUptaneMessageType ::= ENUMERATED {
+		discoveryReq(0),
+		discoveryResp(128),
+		publicKeyReq(1),
+		publicKeyResp(129),
+		manifestReq(2),
+		manifestResp(130),
+		putMetaReq(3),
+		putMetaResp(131),
+		rootVersionReq(4),
+		rootVersionResp(132),
+		putRootReq(5),
+		putRootResp(133),
+		sendFirmwareReq(6),
+		sendFirmwareResp(134),
+		...
+        }
+
+	AKIpUptaneKeyType ::= ENUMERATED {
+		rsa(0),
+		ecdsa(1),
+		...
+	}
+	
+	AKIpUptaneMetadataFormat ::= ENUMERATED {
+		json(0),
+		ber(1),
+		...
+	}
+
+	AKDiscoveryReqPayload ::= SEQUENCE {
+		...
+	}
+
+	AKDiscoveryRespPayload ::= SEQUENCE {
+		hwId OCTET STRING,
+		ecuSerial OCTET STRING,
+		...
+	}
+
+	AKPublicKeyReqPayload ::= SEQUENCE {
+		...
+	}
+
+	AKPublicKeyRespPayload ::= SEQUENCE {
+		type AKIpUptaneKeyType,
+		key OCTET STRING,
+		...
+	}
+
+	AKManifestReqPayload ::= SEQUENCE {
+		...
+	}
+
+	AKManifestRespPayload ::= SEQUENCE {
+		format AKIpUptaneMetadataFormat,
+		manifest ANY DEFINED BY format -- OCTET string for json of ASN.1 sequence (to be defined) for ber
+	}
+	
+	AKImageMeta ::= SEQUENCE {
```

Comment:
Here, `DEFINED BY format` should be `DEFINED BY imageTimestampFormat`

(same issue in other fields and in `AKPutMetaReqPayload`)

Reply:
@lbonn Good catch, thank you.